style.css

```diff
@@ -247,5 +249,4 @@ body {
     display: flex;
     flex-direction: row;
     justify-content: space-between;
-    width: 35%;
 }
```

Comment:
please add the new line in the end of file

Reply:
did

Reply:
but when i saved Visual studio dell new line

Reply:
We need to fix it in the config